### ISSUE TITLE
Feature Request: Support for requiresSession property for Queues

### ISSUE THREAD

**Issue by williamthiago** [terraform-providers/terraform-provider-azurerm#1092]
Hi all,

I guess this property is missing for servicebus queue creation:
https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-sessions

### Terraform Version
Terraform v0.11.6
+ provider.azurerm v1.3.2

### Affected Resource(s)
- azurerm_servicebus_queue

### Terraform Configuration Files
```hcl
resource "azurerm_servicebus_queue" "servicebus" {
  name                = "${var.servicebus["queue_name"]}"
  resource_group_name = "${azurerm_resource_group.resource_group.name}"
  namespace_name      = "${azurerm_servicebus_namespace.servicebus.name}"

  enable_partitioning = true
  # requires_session = true  <--- I need something like this
}
```

**Comment by katbyte** [comment]
Hello @williamthiago,

Thank you for opening this issue and pointing out this missing property 🙂
 
I have added the field and it should be released soon in 1.3.3.

**Comment by williamthiago** [comment]
@katbyte thanks!
